protected/humhub/libs/Helpers.php

```diff
@@ -197,12 +197,12 @@ public static function CheckClassType($className, $type = '')
 
```

Comment:
Gives a `PHP Notice:  Undefined variable: t`

Below fixes this

```php
    /**
     * Checks if the class has this class as one of its parents
     *
     * @param string $className
     * @param string $type
     * @param string $t
     * @return boolean
     */
    public static function CheckClassType($className, $t, $type = '')
    {
        $className = preg_replace('/[^a-z0-9_\-\\\]/i', '', $className);

        if (is_array($type)) {
            foreach ($type as $t) {
                if (class_exists($className) && is_a($className, $t, true)) {
                    return true;
                }
            }
        } else {
            if (class_exists($className) && is_a($className, $t, true)) {
                return true;
            }
        }

        throw new Exception("Invalid class type! (" . $className . ")");
    }
```

Reply:
But then you'll get this error once this is fixed
`is_a() expects parameter 2 to be string, array given `

Reply:
The second is_a check is wrong and should use type parameter. This function also needs some proper doc.

Reply:
Thanks!